core/db/entities.py

```diff
@@ -71,269 +69,236 @@ class Package(BaseModel):
-    status = sqla.Column(sqla.Enum(const.StatusEnum, name="status"), nullable=False)
-    procurement_status = sqla.Column(sqla.Enum(const.ProcurementStatusEnum, name="procurement_status"), nullable=False)
+    submission_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("submission_dim.id"), nullable=False)
+    programme_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("programme_dim.id"), nullable=False)
 
-    # TODO: Should this be nullable? Is Status alone enough in some circumstances?
-    comments = sqla.Column(sqla.String(), nullable=False)
+    question = sqla.Column(sqla.String(), nullable=False)
+    indicator = sqla.Column(sqla.String(), nullable=False)
+    # TODO: nullable as in the data model ss
+    response = sqla.Column(sqla.String(), nullable=True)
+    # TODO: nullable as in the data model ss
+    guidance_notes = sqla.Column(sqla.String(), nullable=True)
 
     # TODO: does this unique index look right?
     # Unique index for data integrity.
     __table_args__ = (
         sqla.Index(
-            "ix_unique_procurement",
-            "construction_contract",
-            "project_id",
+            "ix_unique_funding_question",
+            "submission_id",
+            "programme_id",
             unique=True,
         ),
     )
 
 
-class ProjectProgress(BaseModel):
-    """Stores Project Progress answers."""
+class Project(BaseModel):
+    """Stores Project Entities."""
 
-    __tablename__ = "project_progress"
+    __tablename__ = "project_dim"
+
+    submission_id = sqla.Column(sqla.String(), nullable=False, unique=False)
+    project_id = sqla.Column(sqla.String(), nullable=False, unique=False)
 
-    package_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("package_dim.id"), nullable=False)
+    # TODO: I think this will now be a separate dim?
+    # project_name = sqla.Column(sqla.String(), nullable=False)
 
-    answer_1 = sqla.Column(sqla.String(), nullable=True)
-    answer_2 = sqla.Column(sqla.String(), nullable=True)
-    answer_3 = sqla.Column(sqla.String(), nullable=True)
-    answer_4 = sqla.Column(sqla.String(), nullable=True)
-    answer_5 = sqla.Column(sqla.String(), nullable=True)
-    answer_6 = sqla.Column(sqla.String(), nullable=True)
+    # TODO: This mapping doesn't exist now
+    # package_id: Mapped[int] = sqla.orm.mapped_column(sqla.ForeignKey("package_dim.id"), nullable=True)
 
-    # Unique index for data integrity. There can't be multiple direct fund rows for a single project with
-    # the same date range and direct fund metrics.
+    primary_intervention_theme = sqla.Column(sqla.String(), nullable=False)
+    location_multiplicity = sqla.Column(
+        sqla.Enum(const.MultiplicityEnum, name="project_location_multiplicity"), nullable=False
+    )
+    # TODO: configured as an array of strings. is this ok? can we / should we validate for this?
+    locations = sqla.Column(sqla.ARRAY(sqla.String), nullable=False)
```

Comment:
also, can we please add a field to capture the "Are you providing a GIS map (see guidance) with your return?" column. Should be nullable and a yes/no enum.